File: common/src/main/java/com/jmsoftware/maf/common/websocket/WebSocketMessageResponse.java
package com.jmsoftware.maf.common.websocket;

import lombok.Data;

import java.time.LocalDateTime;

/**
 * Description: WebSocketMessagePayload, change description here.
 *
 * @author Johnny Miller (锺俊), email: johnnysviva@outlook.com, date: 7/15/2021 5:35 PM
 **/
@Data
public class WebSocketMessageResponse {
    private final LocalDateTime timestamp = LocalDateTime.now();
    private String sender;
    private String receiver;
    private Object content;
}

File: common/src/main/kotlin/com/jmsoftware/maf/common/websocket/WebSocketMessageResponse.kt
package com.jmsoftware.maf.common.websocket

import java.time.LocalDateTime

/**
 * # WebSocketMessageResponse
 *
 * Description: WebSocketMessagePayload, change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/16/22 8:41 PM
 */
class WebSocketMessageResponse {
    var timestamp: LocalDateTime = LocalDateTime.now()
    var sender: String? = null
    var receiver: String? = null
    var content: Any? = null
}
